### ISSUE TITLE
Local Upload of ISOs sometimes works 

### ISSUE THREAD

**Issue by Tbaugus44** [apache/cloudstack#8466]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.18.1.0
~~~

##### CONFIGURATION

advanced networking

##### OS / ENVIRONMENT

Cloudstack is running on Ubuntu 22.04.03LTS. The Resource host are running KVM as their hypervisor. 

##### SUMMARY

I have uploaded local iso files successfuly the first time. then i go to upload the same exat iso with a diffrent name and switch the toggle on for featured and Extradable. The iso failes to upload. 

##### STEPS TO REPRODUCE

~~~
Naviagte to images>ISOs>
![image](https://github.com/apache/cloudstack/assets/148808916/75f5d70d-6a4c-4b52-b8c6-0ba02a07e8e5)

~~~

##### EXPECTED RESULTS

~~~
to upload an iso every time 
~~~

##### ACTUAL RESULTS

~~~
it works sometimes 
~~~

**Comment by Tbaugus44** [comment]
[cloud.log](https://github.com/apache/cloudstack/files/13864631/cloud.log)
Here is the cloud.log file from the ssvm

**Comment by kiranchavala** [comment]
@Tbaugus44 Could you please provide the iso names that you tried to upload. From the logs i don't see any major exception

I was unable to reproduce the issue in my lab,

**Comment by Tbaugus44** [comment]
![image](https://github.com/apache/cloudstack/assets/148808916/c1bb3333-e557-4f50-9354-acff556f7446)

**Comment by Tbaugus44** [comment]
We have solved the issue! we will be posting a full git article about our findings and what we did to solve the issue. There were two SSVMs active in the DB. One was destroyed and you couldn't see it from the UI. However, it was causing disconnects thats why we were sometimes able to upload iso and then not be able to. What we did to fix the issue was edit the database column of the ssvm that was destroyed but did not have a destroyed date. we put a date in there and it worked. All local ISOs uploaded just fine over and over again. We will link the official git article here once it's finished.
